### DISCUSSION TITLE
Is there an option for country restriction

### DISCUSSION THREAD

**Discussion by robpanes-space** [visgl/react-google-maps#422] [Q&A] [answered]
Like this

      const request = {
        input: inputValue,
        sessionToken,
        componentResrictions: { country: 'us' },
      };
      const response = await autocompleteService.getPlacePredictions(request);

**Comment by usefulthink** [comment]
Not sure what you are asking about. This library supports the entirety of the maps API in some form, so you can also use the `componentRestriction.country` field for the autocomplete place predictions.

**Reply by robpanes-space** [reply]
@usefulthink 
```
const request = {
    input: inputValue,
    sessionToken,
    componentResrictions: { country: 'us' },
  };
  const response = await autocompleteService.getPlacePredictions(request);
```
but everytime the autocompleteServices get predictions I get this error
![image](https://github.com/visgl/react-google-maps/assets/2166158/f2be5518-9f55-4ba9-a06f-7e7f3287a939)
****
if I remove the restriction: componentResrictions: { country: 'us' } 
it works fine. 

Thanks

**Reply by robpanes-space** [reply, marked as answer]
Oh its a typo, 

componentResrictions -> componentRestrictions.

it works now.
